### PR TITLE
[epsonprojector] several small improvements and formatting changes

### PR DESCRIPTION
**Pull request by mlobstein**
This PR contains several small improvements and formatting changes for epsonprojector similar to those that were done for benqprojector in PR #12866.

### CODE REVIEW

File: bundles/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/discovery/EpsonProjectorDiscoveryService.java [hunk 1]
```diff
@@ -120,17 +129,20 @@ private synchronized void discover() {
...
                         ThingUID thingUid = new ThingUID(THING_TYPE_PROJECTOR_TCP, uid);
                         logger.trace("Creating epson projector discovery result for: {}, IP={}", uid, ipAddress);
-                        thingDiscovered(DiscoveryResultBuilder.create(thingUid).withProperties(thingProperties)
-                                .withLabel("Epson Projector " + uid).withRepresentationProperty(THING_PROPERTY_MAC)
-                                .build());
+                        thingDiscovered(
+                                DiscoveryResultBuilder.create(thingUid).withProperties(thingProperties)
+                                        .withLabel(translationProvider.getText(bundle,
+                                                "thing-type.epsonprojector.discovery.label", "Epson Projector",
+                                                localeProvider.getLocale()) + " " + uid)
```
Comment (jlaur) [inline bundles/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/discovery/EpsonProjectorDiscoveryService.java:144]: I would propose to use a placeholder for the `uid`, so the position is flexible for translations:
```
thing-type.epsonprojector.discovery.label = Epson Projector {0}
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks, looks good! Only a minor suggestion for the now localized discovery label.

File: bundles/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/discovery/EpsonProjectorDiscoveryService.java (hunk 1 above)
Comment (mlobstein) [reply bundles/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/discovery/EpsonProjectorDiscoveryService.java]: fixed

**Review by jlaur — APPROVED** [review]

**Comment by jlaur** [comment]
@mlobstein - thanks! I have removed the enhancement label in order to leave it out of the release notes, since there are no functional changes for the users.

**Pull request merged**